libdrgn/language.h

```diff
@@ -198,6 +200,42 @@ drgn_element_format_object_flags(enum drgn_format_object_flags flags)
 		(flags & DRGN_FORMAT_OBJECT_ELEMENT_TYPE_NAMES) >> 2);
 }
 
+/** Return the drgn language spec corresponding to the given dwfl language */
+static const struct drgn_language *drgn_dwarf_to_language(int dwfl_lang)
+{
+	switch (dwfl_lang) {
+	case DW_LANG_C:
+	case DW_LANG_C89:
+	case DW_LANG_C99:
+	case DW_LANG_C11:
+		return &drgn_language_c;
+	default:
+		return &drgn_default_language;
+	}
+}
+
+/**
+ * Deduce language which corresponds with the CU of the given dwarf die.
+ *
+ * @param[in] die Dwarf_Die used to retrieve the current language.
+ */
+static const struct drgn_language *drgn_language_from_dwarf(Dwarf_Die *die)
```

Comment:
This is only used in `dwarf_info_cache.c`, correct? Let's just move it there.

Also, naming-wise, `drgn_language_from_die` would be a little clearer.